### PR TITLE
Update Files and Uploads page documentation

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3845](https://openedx.atlassian.net/browse/DOC-3845)

Update documentation for the Studio **Files & Uploads** page.

**Note**: I'm planning to update the image so it's less blurry.

### Date Needed (optional)

26 Jan 2018

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: @edx/educator-dahlia 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc 
- [ ] Product review: @sstack22 
- [ ] Partner support: @jaakana

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] http://draft-files-and-uploads-page-feb-2018.readthedocs.io/en/latest/course_assets/course_files.html

### Sandbox (optional)

- [ ] https://studio.stage.edx.org/assets/course-v1:AnalyticsX+dsjen101+2015_T3/

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/shared/course_assets/course_files.rst [hunk 1]
```diff
@@ -4,178 +4,201 @@
 Adding Files to a Course
 ###########################
 
-To use images and other files in your course, you upload the files to your
-course by using Studio, and then you add links to the files in the course
-content. You can also use uploaded files in the bulk email messages that course
-team members can send from the LMS by selecting **Instructor** to access the
-instructor dashboard.
+Adding files, such as images, that you can use in your course has the following
+steps.
```
Comment (efischer19) [inline en_us/shared/course_assets/course_files.rst:8]: Would this look better ending with a `:` to offset the list?

File: en_us/shared/course_assets/course_files.rst [hunk 2]
```diff
@@ -4,178 +4,201 @@
...
 
+.. _The Files and Uploads Page:
+
+*************************
+The Files & Uploads Page
+*************************
+
+You manage most files for your course, including image files, on the **Files &
+Uploads** page. This page lists the files that you upload, along with the
+following information about the files.
```
Comment (efischer19) [inline en_us/shared/course_assets/course_files.rst:47]: Nevermind, looks like there's a pre-established pattern of not using `:` in these cases, disregard the above

File: en_us/shared/course_assets/course_files.rst [hunk 3]
```diff
@@ -4,178 +4,201 @@
...
 file sizes before you add the files to your course. If you have to use files
-that are larger than 50 MB after compression, contact your edX partner manager.
+that are larger than 10 MB after compression, contact your edX partner manager.
 
-If you have video or audio files or large data sets to share with your
-students, use YouTube or another hosting service to host these files. Do not
-use the Files & Uploads page to add these files to your course. If you are an
-edX partner and you need to store large data sets, contact your edX partner
-manager.
+If you have audio files or large data sets to share with your students, do not
```
Comment (efischer19) [inline en_us/shared/course_assets/course_files.rst]: On first read, this seems to be discouraging the use of any audio files ever, which seems strange given that we have an `Audio` filter. Was that the intention?

**Review by efischer19 — APPROVED** [review]
Looks great! The 'Audio' question is the only thing I'm not sure about here

File: en_us/shared/course_assets/course_files.rst [hunk 4]
```diff
@@ -4,178 +4,201 @@
...
 #. Create or locate the file on your computer.
 
    .. note::
-    When URLs are generated for uploaded files, the file name becomes part of
-    the URL and is visible to students when they access the file. Avoid using
-    file names that contain information about the file contents that you do not
-    want to share, such as "Answerkey.pdf".
+     When Studio generates URLs for uploaded files, the file name becomes part
+     of the URL and is visible to students when they access the file. Avoid
+     using file names that contain information about the file contents, such as
```
Comment (grantgoodmanedX) [inline en_us/shared/course_assets/course_files.rst]: I think the phrase "that you do not want to share" is worth keeping here.

**Review by grantgoodmanedX — APPROVED** [review]
👍  This looks fine, with one question.

**Comment by dsjen** [comment]
re: HTML Version, I'm not sure! Does that mean that these orgs would get a separate read the docs? I think this might be more of a @sstack22 question?

**Comment by srpearce** [comment]
@dsjen They would temporarily get a different RTD version—the URL would be something like http://edx.readthedocs.io/projects/**draft_file_uploads_page_updates**/en/latest/course_assets/course_files.html instead of http://edx.readthedocs.io/projects/**edx-partner-course-staff**/en/latest/course_assets/course_files.html. After the feature releases to everyone, we would delete that temporary documentation and everyone would use the regular docs. @sstack22 , your thoughts?

File: en_us/shared/course_assets/course_files.rst [hunk 5]
```diff
@@ -4,178 +4,228 @@
...
 
-To use images and other files in your course, you upload the files to your
-course by using Studio, and then you add links to the files in the course
-content. You can also use uploaded files in the bulk email messages that course
-team members can send from the LMS by selecting **Instructor** to access the
-instructor dashboard.
+Adding files, such as images, that you can use in your course has the following
+steps.
+
+.. image:: ../../../shared/images/AddFiles.png
```
Comment (srpearce) [inline en_us/shared/course_assets/course_files.rst:10]: @fysheets @edx/educator-dahlia @sstack22 I clarified the Studio and web URLs here in this image (scroll down to see the image file), in the intro to this topic, and in the "Use an Uploaded File Inside or Outside the Course" section.

I wonder if we could make this even clearer by changing the names of the buttons in the UI. For example, we could change "Studio" to "In-course URL" and "Web" to "External URL", or similar.

File: en_us/shared/course_assets/course_files.rst [hunk 6]
```diff
@@ -4,178 +4,228 @@
...
+ :alt: A workflow diagram showing the steps to add and show a file in a course
+     or on the internet.
+
+#. The course team uploads a file on the **Files & Uploads** page in Studio.
+#. Studio generates two URLs for the file.
```
Comment (srpearce) [inline en_us/shared/course_assets/course_files.rst:15]: This and the following step explain the difference between Studio and web URLs.

File: en_us/shared/course_assets/course_files.rst [hunk 7]
```diff
@@ -4,178 +4,228 @@
...
+To upload additional files, drag more files into the **Drag and Drop** area of
+the page.
+
 .. _File URLs:
 
-********************************************
-Use File URLs to Reference Uploaded Files
-********************************************
+*************************************************
+Use an Uploaded File Inside or Outside the Course
```
Comment (srpearce) [inline en_us/shared/course_assets/course_files.rst:116]: I divided this section into two sub-sections, one for adding files inside the course, and one for adding files outside the course.

**Review by srpearce — COMMENTED** [review]
To see this topic in context with the image, go to:

http://draft-files-and-uploads-page-feb-2018.readthedocs.io/en/latest/course_assets/course_files.html

**Pull request merged**